README.md

```diff
@@ -1,7 +1,8 @@
 # Dataflow
 
-[![Build](https://github.com/argoproj-labs/argo-dataflow/actions/workflows/build.yml/badge.svg)](https://github.com/argoproj-labs/argo-dataflow/actions/workflows/build.yml)
-[![codecov](https://codecov.io/gh/argoproj-labs/argo-dataflow/branch/main/graph/badge.svg?token=yKtOCXJu1Q)](https://codecov.io/gh/argoproj-labs/argo-dataflow)
+## NOTICE
+Argo Dataflow has been reimplemented in the scope of a broader project focussed on operational data analytics.
```

Comment:
```suggestion
Argo Dataflow has been reimplemented in the scope of a broader project focussed on generic event-driven processing, the foundation for powerful streaming processing semantics.
```

Many users are thinking of using argo-dataflow as a replacement for argo-events. Won't this give a feeling that `numaflow` can only be used for operational analytics? WDYT?

Reply:
Don't want to confuse users as to when to use Events vs Numaflow.
Reworded slightly